examples/accordion/accordion.html

```diff
@@ -259,13 +256,13 @@ <h2 id="rps_label">Role, Property, State, and Tabindex  Attributes</h2>
         <tr>
           <th scope="row"><code>heading</code></th>
           <td></td>
-          <td><code>dt</code></td>
+          <td><code>div</code></td>
           <td>Identifies the element as a heading that serves as an accordion header.</td>
```

Comment:
We can delete this row of table; no ARIA required to make the accordion header into a heading.